### DISCUSSION TITLE
Where can I get the Airsim.exe file?

### DISCUSSION THREAD

**Discussion by Deepakgthomas** [microsoft/AirSim#3419] [Support Q&A] [answered]
I am trying to run different instances of Airsim with Unreal Engine. That is have a different simulation for every instance. In order to do that, I will have to change the Airsim ports. I am following these instructions to have different setting.json files - https://microsoft.github.io/AirSim/settings/, but I cannot figure out where I can get the Airsim.exe file.

**Comment by ahmed-elsaharti** [comment, marked as answer]
Hi @Deepakgthomas , I think the guide assumes that you're running AirSim through a cooked binary. Try using one of the binaries in the [releases section](https://github.com/microsoft/AirSim/releases) or [package your own binary through UE](https://docs.unrealengine.com/en-US/Basics/Projects/Packaging/index.html). If you decide to go through this route, do have a look at #3195.

If you just want to do this through multiple instances of the Unreal Editor you could just edit the main settings.json file (under Documents/AirSim), run the simulation on the first UE instance, edit the same settings.json file then run the 2nd instance. This would probably work since AFAIK once the simulation is run the parameters are grabbed from settings.json and its not accessed again. ie: if you change the settings after running the first instance it won't be affected and the new settings will be picked up by whatever instance u run after the edits.

**Reply by Deepakgthomas** [reply]
Thank you @ahmed-elsaharti ! I have a quick question, regarding setting up the port. How does one know which port to use? For instance, after starting my first simulation I'll use the port 41451 (the default one). What port should I use for my second simulation on the same PC?

**Reply by ahmed-elsaharti** [reply]
Any open port should be good I think. But technically you'd want to choose something according to whats mentioned in #3395
